test/paper-input.html

```diff
@@ -196,23 +196,51 @@
         input = fixture('basic');
       });
 
-      test('focus/blur events fired on host element', function() {
-        var nFocusEvents = 0;
-        var nBlurEvents = 0;
+      // At the moment, it is very hard to correctly fire exactly
+      // one focus/blur events on a paper-input. This is because
+      // when a paper-input is focused, it needs to focus
+      // its underlying native input, which will also fire a `blur`
+      // event.
```

Comment:
Oh! I just split them up into 2 different tests! There's the equivalent `blur events fired on host element` tests below.